ethdb/privateapi/ethbackend.go

```diff
@@ -896,6 +897,10 @@ func (s *EthBackendServer) Peers(ctx context.Context, _ *emptypb.Empty) (*remote
 	return s.eth.Peers(ctx)
 }
 
+func (s *EthBackendServer) AddPeer(ctx context.Context, url *remote.AddPeerRequest) (*remote.AddPeerReply, error) {
+	return s.eth.AddPeer(ctx, url)
```

Comment:
```suggestion
func (s *EthBackendServer) AddPeer(ctx context.Context, req *remote.AddPeerRequest) (*remote.AddPeerReply, error) {
	return s.eth.AddPeer(ctx, req)
```

why don't you use `req`?

Reply:
Fixed